tests.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
```

Comment:
its common to have tests files inside a `tests/` directory. Also the files should have discriptive names indicating which parts of the system they test

Reply:
I have the tests in the tests.py file and it in its turn runs functions that return a boolean to comply with the assert command in the tests.py file, also I changed the function names to test_ and the test name to work as it should with pytest.

Reply:
@ifireball Is the answer satisfies your ask?